app/profiles/landlord/page.tsx

```diff
@@ -0,0 +1,91 @@
+import { createClient } from "@/utils/supabase/server"
+import { cookies } from "next/headers"
+import { NextPage } from "next"
+
+const landlordProfilePage: NextPage = async ({
+    searchParams
+}: {
+    searchParams?: {
+        landlordId?: string,
+    }
+}) => {
+    // check if a landlord id was provided
+    const landlordId = searchParams?.landlordId
+    if (!landlordId) return (
+        <div>
+            <h1>ERROR: No Landlord Id provided</h1>
+        </div>
+    )
+
+    // set up the supabase client
+    const cookieStore = cookies()
+    const supabase = createClient(cookieStore)
+
+    // check if a landlord with the provided id exists and get their info
+    const { data: landlordData, error: landlordError } = await supabase
+        .from('user_profiles')
```

Comment:
If we want to be able to show that to all users, then we should be storing it in the landlord profiles, as the user profiles are private as per the schema plan:
<img width="1196" alt="Screenshot 2024-02-22 at 00 54 08" src="https://github.com/CM20254-Group-7/Rental_Review/assets/74842299/ad227b9f-dab9-458d-b96a-eab6ccd0c623">

I also believe we have dropped the first and last name from the user profiles, as we are not collecting it from users who are not landlords.

it might also be beneficial to have a separate data point for the landlords public facing email if they want to show a different one to the one they use to login

Reply:
I also suggested there that we might want to make the landlord's real name private and give them a username for the public but i don't think we want to do it that way